terraform/development/main.tf

```diff
@@ -26,6 +26,24 @@ provider "aws" {
   }
 }
 
+provider "aws" {

```

Comment:
These aren't in alphameric order because I thought it made sense to keep our default provider at the top.